lib/galaxy/tools/search/__init__.py

```diff
@@ -25,6 +25,7 @@
 from galaxy.web.framework.helpers import to_unicode
 
 log = logging.getLogger(__name__)
+HTML_CLEANER = re.compile("<[^<]+?>")
```

Comment:
poor man's beautiful soup/lxml, please offer better alternatives if you have

Reply:
Do you have any experience with its speed?

Reply:
so it does not really beahve as I want it to:

```python
>>> import bleach
>>> bleach.clean('test<strong>sdsd</strong>', strip=True)
u'test<strong>sdsd</strong>'
>>> bleach.clean('test<span></span>', strip=True)
u'test'
```

Reply:
it seems I need to define an empty list of allowed tags, then it works well, commit inc.